app/client-app/src/services/generate-combination.js

```diff
@@ -0,0 +1,26 @@
+import adjectives from "./adjectives";
+import animals from "./animals";
+
+const generateCombination = (
```

Comment:
this function (and file) need a better name. Generate a combination of what?

Honestly, the file should probably be called `gfycat-style-urls.js` and the function `generateGfyCatStyleUrl()` or something...

Also, we need to retain attribution to the original package to comply with the license